tools/CoqMakefile.in

```diff
@@ -389,16 +389,20 @@ optfiles: $(if $(DO_NATDYNLINK),$(CMXSFILES))
 .PHONY: optfiles
 
 # FIXME, see Ralf's bugreport
-quick: $(VOFILES:.vo=.vio)
+# quick is deprecated, now renamed vio
```

Comment:
IIRC make provides `$(warning "text")` so you should be able to put this comment into the `quick:` target and have it displayed